lib/node_modules/@stdlib/stats/base/dnanvariancech/docs/types/index.d.ts

```diff
@@ -68,21 +68,21 @@ interface Routine {
 * @param N - number of indexed elements
 * @param correction - degrees of freedom adjustment
 * @param x - input array
-* @param stride - stride length
+* @param strideX - stride length
 * @returns variance
 *
 * @example
 * var Float64Array = require( '@stdlib/array/float64' );
 *
-* var x = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
+* var x = new Float64Array( [ 1.0, -2.0, 2.0 ] );
```

Comment:
```suggestion
* var x = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
```
copy and paste error